neonvm/runner/container-mgr/crictl.go

```diff
@@ -0,0 +1,146 @@
+package main
+
+// crictl abstraction and commands
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+
+	"go.uber.org/zap"
+)
+
+type Crictl struct {
+	endpoint string
+}
+
+// Pods calls 'crictl pods' and, if successful, returns the parsed output
+//
+// This command lists all running pods.
+func (c *Crictl) Pods(logger *zap.Logger) (*crictlPods, error) {
```

Comment:
Is there a reason that `crictPods` is not exported? It's a bit weird that an exported function returnes an unexported type.

Reply:
Yeah, fair - updated in f1f564275337b517b657a23be44121af6748b080.